test/teslamate/vehicles/vehicle/charging_test.exs

```diff
@@ -64,6 +66,7 @@ defmodule TeslaMate.Vehicles.Vehicle.ChargingTest do
                       date: _,
                       charge_energy_added: 0.3,
                       rated_battery_range_km: 4.83,
+                      charge_rate: 10,
```

Comment:
these values should set to realisitc values

Reply:
As before, realistic value of preconditioning lowering requirement

Reply:
as before